charts/cadence/templates/grafana-deployment.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: apps/v1
```

Comment:
It's not common practice to include a separate copy of prometheus stack. Most K8s users will have those components installed via another chart.